.github/workflows/script/PrLabeled.ps1

```diff
@@ -0,0 +1,50 @@
+param(
+    [System.String]
+    $LabelName,
+    [System.String]
+    $PrUrl
+)
+Write-Host "Processing PR $PrUrl with label $LabelName"
+
+$CommentDict = @{
+    "Breaking Change Release" = "Azure PowerShell breaking change policy, please refer to [Breaking Change Definition](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/breaking-change/breaking-changes-definition)"
+    "Breaking Change - attribute" = "Please check, the attribute is missing in the breaking change or the attribute setting is incorrect, please refer to [Breaking Changes Attribute Help](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/breaking-change/breaking-changes-attribute-help)"
```

Comment:
We need to revise the breaking change doc on eng hub. We should add an overview page, add a breaking change policy page, and also include breaking change doc in autorest. I'll add an ADO task in this sprint.